src/EnergyPlus/HeatRecovery.cc

```diff
@@ -5386,6 +5403,28 @@ namespace HeatRecovery {
         }
     }
 
+    bool GetHeatRecoveryHXNominalFlowSizingFlag(int const OASysNum) 
+    {
+        bool HXNominalFlowSizingFlag;
+        HXNominalFlowSizingFlag = false;
+        if (OASysNum == 0) {
+            return HXNominalFlowSizingFlag;
+        }
+        auto &thisOASys = DataAirLoop::OutsideAirSys(OASysNum);
+        if (thisOASys.OAControllerIndex > 0) {
+            auto &thisOAController(MixedAir::OAController(thisOASys.OAControllerIndex));
+            if (thisOAController.ControllerType_Num == MixedAir::ControllerOutsideAir) {
+                if (thisOAController.Econo == MixedAir::NoEconomizer || thisOAController.EconBypass ||
+                    thisOAController.Lockout == MixedAir::LockoutWithHeatingPossible ||
+                    thisOAController.Lockout == MixedAir::LockoutWithCompressorPossible ||
+                    (thisOAController.Lockout == MixedAir::NoLockoutPossible &&
+                     thisOAController.HeatRecoveryBypassControlType == DataHVACGlobals::BypassWhenWithinEconomizerLimits)) {
+                    HXNominalFlowSizingFlag = true;
+                }
+            }
+        }
+        return HXNominalFlowSizingFlag;
+    }
 } // namespace HeatRecovery
```

Comment:
Added new function that set the value of SizeHRHXtoMinFlow flag depending on the OA controller inputs. Various combination of economizer inputs such economizer type, bypass control, and lockout status determine the flag value.

Reply:
If `thisOAController.Econo == MixedAir::NoEconomizer`, I don't think it will ever send a bypass signal, so I think this should be the first test and if it's true, set the function to false and skip the rest.
Why do the Lockout options apply here?